workflows/README.md

```diff
@@ -1 +1,58 @@
+# Getting started with Docker
+
+If you want to run these protocols interactively you will need to install Docker on your system. You can find more detailed instructions [here](https://docs.docker.com/install/). Alternatively we've developed snakefiles for automation that do not require installation. You can find those instructions below. 
+
+#### First, update your machine
+```
+sudo apt-get -y update && \
+sudo apt-get -y install python-pip \
```

Comment:
We should be careful with respect to Python and the version of Python we're using. Python in particular is a fast-changing language and a year from now the versions we're using could be obsolete. (Python 2 is going to be officially obsolete in 2 years.)

I like the [pyenv](https://github.com/pyenv/pyenv) tool for maintaining Python versions, since it is really extensive and provides anaconda, miniconda, plain python, pypy, and others. 

Using Python from a container might also be possible, and could give you control over the version of Python you're using. But control over the version number is important to keeping the pipelines reliable for the years+ scale.

Reply:
Addressed with commits below, which add pyenv to the instructions.